package.json

```diff
@@ -62,13 +62,14 @@
         "jest": "^24.8.0",
         "jest-dom": "^3.5.0",
         "jest-junit": "^6.4.0",
+        "jest-watch-typeahead": "^0.4.2",
         "lint-staged": "^8.0.4",
         "lodash": "^4.17.13",
         "prettier": "^1.15.2",
         "progress-bar-webpack-plugin": "^1.11.0",
-        "react": "16.8.6",
+        "react": "^16.12.0",
```

Comment:
I've needed async act to fix tests and this has been introduced in 16.9, so had to update those.